public/index.html

```diff
@@ -85,51 +89,48 @@ <h2 class="alt">Hi! I'm <strong>Prologue</strong>, a <a href="http://html5up.net
 								<h2>Portfolio</h2>
 							</header>
 
-							<p>Vitae natoque dictum etiam semper magnis enim feugiat convallis convallis
-							egestas rhoncus ridiculus in quis risus amet curabitur tempor orci penatibus.
-							Tellus erat mauris ipsum fermentum etiam vivamus eget. Nunc nibh morbi quis
-							fusce hendrerit lacus ridiculus.</p>
+							<p>With a background in athletic training, I am excited to be able to help people through technology.</p>
 
 							<div class="row">
 								<div class="col-4 col-12-mobile">
 									<article class="item">
 										<a href="#" class="image fit"><img src="images/pic02.jpg" alt="" /></a>
 										<header>
-											<h3>Ipsum Feugiat</h3>
+											<h3>Future Project 1</h3>
```

Comment:
I recommend adding more information here. Perspective employers are very interested in your projects so far.